tests/lib/tools/snapd-state

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash -e
+
+show_help() {
+    echo "usage: check-state <jq-filter> <comparison> <expected-res> <error-message>"
+    echo "       print-state <jq-filter>"
+    echo "       change-snap-channel <snap-name> <channel>"
+    echo "       force-autorefresh"
+    echo "       prevent-autorefresh"
+    echo "       wait-for-autorefresh <last-change-id>"
```

Comment:
This should be ` wait-for-autorefresh <snap-name> <last-change-id>`?

Reply:
yes, thanks for the review